src/amo/reducers/landing.js

```diff
@@ -0,0 +1,36 @@
+import {
+  LANDING_GET,
+  LANDING_LOADED,
+  LANDING_FAILED,
+} from 'core/constants';
+
+
+export const initialState = {
```

Comment:
aha, I see it now on this line: https://github.com/mozilla/addons-frontend/pull/1439/files#diff-cba04c1c93442d633523fc29d40f97e0R26